docs/source/tutorials/multiannotator.ipynb

```diff
@@ -0,0 +1,555 @@
+    "    sizes=[80, 40, 40],\n",
+    "    num_annotators=50,\n",
+    "    seed=SEED,  # set to None for non-reproducible randomness\n",
+    "):\n",
+    "    np.random.seed(seed=SEED)\n",
+    "\n",
+    "    m = len(means)  # number of classes\n",
+    "    n = sum(sizes)\n",
+    "    local_data = []\n",
+    "    labels = []\n",
+    "\n",
+    "    for idx in range(m):\n",
+    "        local_data.append(\n",
+    "            np.random.multivariate_normal(mean=means[idx], cov=covs[idx], size=sizes[idx])\n",
+    "        )\n",
+    "        labels.append(np.array([idx for i in range(sizes[idx])]))\n",
+    "    X_train = np.vstack(local_data)\n",
+    "    true_labels_train = np.hstack(labels)\n",
+    "\n",
+    "    # Compute p(true_label=k)\n",
+    "    py = np.bincount(true_labels_train) / float(len(true_labels_train))\n",
+    "    \n",
+    "    noise_matrix_better = generate_noise_matrix_from_trace(\n",
+    "        m,\n",
+    "        trace=0.9 * m,\n",
+    "        py=py,\n",
+    "        valid_noise_matrix=True,\n",
+    "        seed=seed,\n",
+    "    )\n",
+    "    \n",
+    "    noise_matrix_worse = generate_noise_matrix_from_trace(\n",
+    "        m,\n",
+    "        trace=0.5 * m,\n",
+    "        py=py,\n",
+    "        valid_noise_matrix=True,\n",
+    "        seed=seed,\n",
+    "    )\n",
+    "\n",
+    "    # Generate our noisy labels using the noise_matrix for specified number of annotators.\n",
+    "    s = pd.DataFrame(\n",
+    "        np.vstack(\n",
+    "            [\n",
+    "                generate_noisy_labels(true_labels_train, noise_matrix_better)\n",
+    "                if i < num_annotators - 5\n",
+    "                else generate_noisy_labels(true_labels_train, noise_matrix_worse)\n",
+    "                for i in range(num_annotators)\n",
+    "            ]\n",
+    "        ).transpose()\n",
+    "    )\n",
+    "\n",
+    "    # Each annotator only labels approximately 10% of the dataset\n",
+    "    # (unlabeled points represented with NaN)\n",
+    "    s = s.apply(lambda x: x.mask(np.random.random(n) < 0.9))\n",
+    "    s.dropna(axis=1, how=\"all\", inplace=True)\n",
+    "\n",
+    "    row_NA_check = pd.notna(s).any(axis=1)\n",
+    "\n",
+    "    return {\n",
+    "        \"X_train\": X_train[row_NA_check],\n",
+    "        \"true_labels_train\": true_labels_train[row_NA_check],\n",
+    "        \"multiannotator_labels\": s[row_NA_check].reset_index(drop=True),\n",
+    "    }\n",
+    "\n",
+    "data_dict = make_data()\n",
+    "\n",
+    "X = data_dict[\"X_train\"]\n",
+    "multiannotator_labels = data_dict[\"multiannotator_labels\"]\n",
+    "true_labels = data_dict[\"true_labels_train\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e95d31b2",
+   "metadata": {},
+   "source": [
+    "For this tutorial we are using a toy dataset that has 50 annotators and 160 examples. There are three possible classes, `0`, `1` and `2`.\n",
```

Comment:
What do think about us manually ensuring the last 5 examples in the dataset also have wrong consensus labels, by purposefully ensuring:
each of these examples only has one annotation, and this annotation is wrong.

This way we could better demonstrate the utility of the `consensus_quality_score`.  And the final hidden cell of this notebook could include a check that these bottom 5 examples receive low-ish consensus_quality_score